### PR TITLE
Add a plugin to use Cactus Comments

### PR DESCRIPTION
**Pull request by pieterdavid**
This plugin relies on the changes in https://github.com/getnikola/nikola/pull/3546 to add Cactus Comments as comment system, so it should be reviewed as part of that, or after the PR is accepted.

### CODE REVIEW

File: v8/cactuscomments/cactuscomments.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+from __future__ import unicode_literals
+from nikola.plugin_categories import ConfigPlugin
+import nikola.utils
+
+class CactusComments(ConfigPlugin):
+    """Use Cactus Comments https://cactus.chat/ as a comment system"""
+    def set_site(self, site):
+        site.template_hooks['extra_head'].append(
+                '<script type="text/javascript" src="https://latest.cactus.chat/cactus.js"></script>'
```
Comment (Kwpolska) [inline v8/cactuscomments/cactuscomments.py:31]: I think the JS script should go in the `comment_form` macro in templates,.

File: v8/cactuscomments/cactuscomments.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from __future__ import unicode_literals
+from nikola.plugin_categories import ConfigPlugin
```
Comment (Kwpolska) [inline v8/cactuscomments/cactuscomments.py:24]: We could also add a `CommentSystemPlugin` for this case (I’ll handle this someday, when both PRs are merged.)

File: v8/cactuscomments/cactuscomments.py (hunk 1 above)
Comment (pieterdavid) [reply v8/cactuscomments/cactuscomments.py]: Thanks! Indeed, it's only the stylesheet that needs to be here

**Review by Kwpolska — APPROVED** [review]

**Comment by Kwpolska** [comment]
@pieterdavid I merged a bit too early, the plugin was missing a `README.md`. I added in a very simple one to unbreak the build — but please write a nice README and make a new PR.

**Comment by pieterdavid** [comment]
@Kwpolska done in https://github.com/getnikola/plugins/pull/383 - thanks a lot for all the help with getting this included!

**Pull request merged**